app/uk/gov/hmrc/pdsauthcheckerapi/controllers/AuthorisationController.scala

```diff
@@ -20,10 +20,15 @@ import javax.inject._
 import play.api.Configuration
 import play.api.libs.json.Json
 import play.api.mvc._
+import uk.gov.hmrc.pdsauthcheckerapi.models
+import uk.gov.hmrc.pdsauthcheckerapi.models.PdsAuthRequest
+import uk.gov.hmrc.pdsauthcheckerapi.services.PdsService
 import uk.gov.hmrc.play.bootstrap.backend.controller.BackendController
 
+import scala.concurrent.{ExecutionContext, Future}
+
 @Singleton
-class AuthorisationController @Inject()(cc: ControllerComponents, config: Configuration) extends BackendController(cc)  {
+class AuthorisationController @Inject()(cc: ControllerComponents, config: Configuration, pdsService: PdsService)(implicit ec: ExecutionContext) extends BackendController(cc)  {
 
   private val supportedAuthTypes: Set[String] = config.get[String]("auth.supportedTypes").split(",").map(_.trim).toSet
```

Comment:
Can we refactor this into either an action that can run before we enter the controller? I'd be happy with a ticket for the work rather than refactoring it now